cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -286,8 +286,8 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 		match call {
 			RuntimeCall::System(frame_system::Call::set_storage { items })
 				if items.iter().all(|(k, _)| {
-					k.eq(&bridging::XcmBridgeHubRouterByteFee::key()) |
```

Comment:
This is certainly a bug, its still not relevant to the issue in the beacon light client. I suggest updating the PR description,  saying you added some miscellaneous unrelated fixes.

Reply:
I reverted this change to keep the PR as focused as possible. I added the change in this PR: https://github.com/paritytech/polkadot-sdk/pull/3761